### PR TITLE
[SDK] Fix no exception when using `set_env_from_file` with non-existent file

### PR DESCRIPTION
**Pull request by quaark**
Fixes https://jira.iguazeng.com/browse/ML-2384

Raise `MLRunNotFoundError ` when using `set_env_from_file` if file doesn't exist

### CODE REVIEW

File: mlrun/__init__.py [hunk 1]
```diff
@@ -194,6 +194,8 @@ def set_env_from_file(env_file: str, return_dict: bool = False):
     :param return_dict: set to True to return the env as a dict
     :return: None or env dict
     """
+    if not path.isfile(env_file):
```
Comment (yaronha) [inline mlrun/__init__.py:197]: need to add: `env_file = os.path.expanduser(env_file)` to resolve `~/` etc.

**Comment by yaronha** [comment]
worth adding the `env_file = os.path.expanduser(os.environ["MLRUN_ENV_FILE"])` to this as well:

https://github.com/mlrun/mlrun/blob/4403730d0ecd1e8483d52585dd5b326258050fa8/mlrun/config.py#L798

**Review by Tankilevitch — APPROVED** [review]
Looks like there is bug in the docs which states that user need to set `MLRUN_SET_ENV_FILE` rather than `MLRUN_ENV_FILE` can you please add that fix in the PR?

File: mlrun/__init__.py [hunk 2]
```diff
@@ -194,6 +194,9 @@ def set_env_from_file(env_file: str, return_dict: bool = False):
     :param return_dict: set to True to return the env as a dict
     :return: None or env dict
     """
+    env_file = path.expanduser(env_file)
+    if not path.isfile(env_file):
+        raise MLRunInvalidArgumentError(f"env file {env_file} does not exist")
```
Comment (Tankilevitch) [inline mlrun/__init__.py:199]:
```suggestion
        raise MLRunNotFoundError(f"env file {env_file} does not exist")
```

**Review by Tankilevitch — CHANGES_REQUESTED** [review]

**Pull request merged**